lib/text/vtt_text_parser.js

```diff
@@ -437,6 +438,46 @@ shaka.text.VttTextParser = class {
     }
   }
 
+  /**
+   * Converts karaoke style tag to be valid for xml parsing
+   * For example,
+   * input: Text <00:00:00.450> time <00:00:01.450> 1
+   * output: <00:00:01.450><00:00:00.450>Text </00:00:00.450>
+   *         time </00:00:01.450> 1
+   *
+   * Returns original payload if invalid tag is found.
```

Comment:
This doesn't seem to be the case?
This will, for example, handle `Text <00:00:00.450 time <00:00:01.450> 1` by just ignoring the first tag, effectively. But it will modify the second tag, so that's not just returning the original payload.
I assume this is a copy-paste error from the description of `replaceColorPayload_`.